src/shared/components/oncoprint/SortUtils.ts

```diff
@@ -302,6 +303,42 @@ export function getClinicalTrackSortComparator(track: ClinicalTrackSpec) {
     };
 }
 
+export function getClinicalTrackInitialSortDirection(
+    track: ClinicalTrackSpec
+): 0 | 1 | -1 {
+    let initialSortOrder: 0 | 1 | -1 = 0;
```

Comment:
would be better to just use string constants think

Reply:
added enum `SortOrder`